libnavui-lifecycle/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
```

Comment:
We don't use anything from this plugin and it is deprecated anyway.

Reply:
copy paste from kotlin-android-extensions `libnavui-speedlimit` so we should delete there too

Reply:
Removing other places https://github.com/mapbox/mapbox-navigation-android/pull/5183